build.gradle

```diff
@@ -3,7 +3,7 @@ task wrapper(type: Wrapper) {
 }
 
 allprojects {
-    version = "1.0.1"
+    version = "1.0.2"
```

Comment:
should do this in a separate PR. i.e. this PR adds the feature, then separately we decide if we are going to up the version for a release